index.html

```diff
@@ -1,14 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>LAMBDA</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap" rel="stylesheet">
+  <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+  <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>LAMBDA - healthy food restaurant</title>
 </head>
+
 <body>
-    <h1>LAMBDA restaurant</h1>
-    
+  <div class="main-wrapper">
+    <header class="main-header">
+      <a class="logo" href="#">Lambda</a>
+      <nav>
+        <ul class="nav">
+          <li>
+            <a class="nav-link is-active">Home</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#">About</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#">Igredients</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#">Menu</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#">Reviews</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#">Reservations</a>
+          </li>
+        </ul>
+      </nav>
+      <ul class="social-links">
+        <li>
+          <a class="social-link" href="https://twitter.com" target="_blank">
+            <ion-icon class="social-icon" name="logo-twitter"></ion-icon>
+          </a>
+        </li>
+        <li>
+          <a class="social-link" href="https://youtube.com" target="_blank">
+            <ion-icon class="social-icon" name="logo-youtube"></ion-icon>
+          </a>
+        </li>
+        <li>
+          <a class="social-link" href="https://facebook.com" target="_blank">
+            <ion-icon class="social-icon" name="logo-facebook"></ion-icon>
+          </a>
+        </li>
+      </ul>
+    </header>
+    <main>
+      <section class="promo-section">
+        <p class="promo-slogan">The right ingredients<br>for the right food</p>
```

Comment:
Since it's a heading, some `h` tag should be used.
```suggestion
        <h2 class="promo-slogan">The right ingredients<br>for the right food</h2>
```
`h2` because `h1` tag has meaning for search engines and should be used to match such kind of goals